src/containers/Guide/GuideAccounts/index.tsx

```diff
@@ -20,22 +20,21 @@ const GuideAccounts: FC = () => {
       <DocImage alt="accounts simple" maxWidth={600} src={AccountsSimple} />
 
       <p>
-        When users first download an account manager, it will take the user through the process of creating a{' '}
-        <strong>key pair</strong>. A key pair consists of two keys, a <strong>signing key</strong> an d an{' '}
-        <strong>account number</strong>.
+        When users first download an account manager, it guides them through creating a <strong>key pair</strong>. A key
+        pair comprises two keys, a <strong>signing key</strong> and an <strong>account number</strong>.
       </p>
 
       <DocImage alt="key pair" maxWidth={380} src={KeyPair} />
 
       <p>
         The account number of an account is like a username, but is unique across the entire network (rather than a
-        single website). The use of signing keys secures all accounts. Signing keys can prove ownership of coins. They
-        are also used in the process of "signing" a block. This allows for the ability of users to create secure
-        transactions that the network can further validate without revealing the signing key itself. The signing key
-        should always remain private to the account owner.
+        single website). Using signing keys secures all accounts and prove ownership of coins. Signing keys are also
+        used in the process of "signing" a block. This allows users to create secure transactions that the network can
+        further validate without revealing the signing key itself. The signing key must always remain private to the
+        account owner.
```

Comment:
First and second sentence are too short - this would work if this was a bulleted list. Also, passive voice is generally avoided (however, I kept it in many spots to bind with Bucky's original style + no passive voice would mean anthropomorphism, like the network does this or determines that. So, you will see that in many spots passive voice was updated (see I do it too, lol). It's not meant to mean less formal, it just follows a general principle like: https://developers.google.com/style/voice

Reply:
"This allows for the ability" although nice and formal readability is the issue. The general idea is to be more readable, with less words